ui/shared/components/form/ReduxFormWrapper.jsx

```diff
@@ -116,35 +146,25 @@ class ReduxFormWrapper extends React.Component {
     } else if (this.props.submitFailed) {
       saveStatus = RequestStatus.ERROR
     }
+
+    // redux-form does not support throwing submission warnings so this is a work around
+    const submissionWarnings = this.props.warning || (this.props.error && this.props.error.map(error => error.warning).filter(warning => warning))
```

Comment:
I was initially in favor of warnings but in retrospect there's no good way to fit them into a dialog like this, so would vote for getting rid of them - the error message when ignoreInvalidItems=false seems sufficient, and then the form can just close on submit when ignoreInvalidItems=true.